application/backend/pyproject.toml

```
@@ -24,6 +24,7 @@ dependencies = [
     "datumaro[experimental] @ git+https://github.com/open-edge-platform/datumaro.git@develop",
     "scikit-learn~=1.7.0",
     "faster-coco-eval~=1.6.8",
+    "scikit-multilearn>=0.2.0",
```

```suggestion
    "scikit-multilearn~=0.2.0",
```
Even if this package hasn't been updated for a long time, it's better to use `~=` than `>=` to avoid problems if they release an API-breaking 1.0 version.